lib/cli.js

```diff
@@ -461,6 +474,18 @@ module.exports = (argv /*: string[]*/) /*: Promise<void>|void*/ => {
       );
     })
     .then(options => {
+      if (cli.flags.printConfig) {
+        return printConfig(options)
+          .then(config => {
+            process.stdout.write(JSON.stringify(config, null, "  "));
+          })
+          .catch((err /*: { stack: any, code: any }*/) => {
+            console.log(err.stack); // eslint-disable-line no-console
```

Comment:
Maybe we need to create a new function to catch errors, not just copy these lines.